Zend/zend_partial.c

```diff
@@ -341,8 +341,24 @@ static HashTable *zend_partial_get_gc(zend_object *obj, zval **table, int *n)
 {
 	zend_partial *partial = (zend_partial *)obj;
 
-	*table = Z_TYPE(partial->This) == IS_OBJECT ? &partial->This : NULL;
-	*n = Z_TYPE(partial->This) == IS_OBJECT ? 1 : 0;
+    if (!partial->argc) {
+        *table = Z_TYPE(partial->This) == IS_OBJECT ? &partial->This : NULL;
+	    *n = Z_TYPE(partial->This) == IS_OBJECT ? 1 : 0;
+    } else {
+        zend_get_gc_buffer *buffer = zend_get_gc_buffer_create();
+        
+        if (Z_TYPE(partial->This) == IS_OBJECT) {
+            zend_get_gc_buffer_add_zval(buffer, &partial->This);
+        }
+        
+        for (uint32_t arg = 0; arg < partial->argc; arg++) {
+            if (Z_OPT_REFCOUNTED(partial->argv[arg])) {
```

Comment:
The REFCOUNTED check is already part of zend_get_gc_buffer_add_zval.

Reply:
Removed in 7718fe0d3d626597c4810b5872818969068297ac